src/ui/FacetValueSuggestions/FacetValueSuggestions.ts

```diff
@@ -131,7 +133,25 @@ export class FacetValueSuggestions extends Component {
      */
     templateHelper: ComponentOptions.buildCustomOption<(row: IFacetValueSuggestionRow, omnibox: Omnibox) => string>(() => {
       return null;
-    })
+    }),
+    /**
+     * Specifies if the field to suggest is a "Category field", similar to the format used by the [`CategoryFacet`]{@link CategoryFacet} component.
```

Comment:
small suggestion:
if -> whether
Also, is the format merely 'similar' to that used by the category facet, or is it exactly the same? If it's the same, I'd go with:
..., similar to the format used by the ... -> ..., i.e., it has the format used by the ...
If it's just similar, let me know where I can get some details and I'll try to suggest something :)

Reply:
No you are right, it's exactly the same format.